projects/plugins/crm/includes/ZeroBSCRM.AJAX.php

```diff
@@ -758,7 +758,7 @@ function ZeroBSCRM_get_quote_template() {
 			}
 			if ( isset( $_POST['quote_fields']['zbscq_date'] ) && ! empty( $_POST['quote_fields']['zbscq_date'] ) ) {
 				$sanitized_date = sanitize_text_field( wp_unslash( $_POST['quote_fields']['zbscq_date'] ) );
-				$quote_date     = wp_date( get_option( 'date_time' ), strtotime( $sanitized_date ) );
+				$quote_date     = jpcrm_uts_to_date_str( strtotime( $sanitized_date ), get_option( 'date_format' ) );
```

Comment:
Note that this function defaults to the WP format, so you could just do this (not a blocker):

```suggestion
				$quote_date     = jpcrm_uts_to_date_str( strtotime( $sanitized_date ) );
```

Reply:
Also, `strtotime()` may also introduce timezone issues.